src/Heading.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+function Heading(props) {
+  const Heading = <header className="App-header">CYF Hotel</header>;
+  return Heading;
+}
+
+export default Heading;
```

Comment:
Pro tip:

In this case, when the component is just one line, you could do this way.
```js
function Heading(props) {
  return <header className="App-header">CYF Hotel</header>;
}
```

Reply:
Another point is. Are you planning to receive any prop in here? If you don't want to make this component dynamic and reusable, you can just take the props of your function declaration.

You can do:
```js
function Heading() {
```

But, maybe would be better to receive the description as a prop. This way you could reuse this component in other parts of your app and provide different texts.